packages/components/src/pages/Quickstart.vue

```diff
@@ -0,0 +1,571 @@
+<template>
+  <div class="qs">
+    <div class="qs-container">
+      <div class="qs-head">
+        <div class="qs-head__btn-wrap">
+          <button
+            class="qs-head__btn"
+            type="button"
+            v-if="currentStep !== 1"
+            @click="prevStep"
+          >
+            <StepLeftIcon class="qs-head__btn-icon" />
+          </button>
+        </div>
+        <span class="qs-head__step"
+          >step {{ currentStep }}/{{ stepsCount }}</span
+        >
+        <div class="qs-head__btn-wrap">
+          <button
+            class="qs-head__btn"
+            type="button"
+            v-if="currentStep !== stepsCount"
+            @click="nextStep"
+          >
+            <StepRightIcon class="qs-head__btn-icon" />
+          </button>
+        </div>
+      </div>
+      <section class="qs-step" v-if="currentStep === 1">
+        <div class="qs-step__head">
+          <h1 class="qs-title">Install AppMap Agent</h1>
+          <select class="qs-select" v-model="language">
```

Comment:
Props should not be edited from within the component or its children. `vmodel="language"` will end up writing to the prop value and raising a console error.